models/product-type/src/transformers.ts

```diff
@@ -1,17 +1,26 @@
+import * as AttributeDefinition from '@commercetools-test-data/attribute-definition';
 import { Transformer } from '@commercetools-test-data/core';
 import type { TProductType, TProductTypeGraphql } from './types';
 
 const transformers = {
   default: Transformer<TProductType, TProductType>('default', {
-    buildFields: ['attributes'],
+    buildFields: ['attributes', 'createdBy', 'lastModifiedBy'],
   }),
   rest: Transformer<TProductType, TProductType>('rest', {
-    buildFields: ['attributes'],
+    buildFields: ['attributes', 'createdBy', 'lastModifiedBy'],
   }),
   graphql: Transformer<TProductType, TProductTypeGraphql>('graphql', {
-    buildFields: ['attributes'],
+    buildFields: ['createdBy', 'lastModifiedBy'],
     addFields: () => ({
-      __typename: 'ProductType',
+      __typename: 'ProductTypeDefinition',
+      // should this pattern be generalized?
+      attributeDefinitions: {
+        limit: 0,
+        offset: 0,
+        total: 0,
```

Comment:
These hardcoded values aren't 100% accurate given they should reflect where the user is in pagination and how many results there are.